modules/KalturaSupport/resources/mw.KWidgetSupport.js

```diff
@@ -314,10 +314,18 @@ mw.KWidgetSupport.prototype = {
 							var match = serviceUrl.match( regExp );
 
 							if (match) {
-								serviceUrl = serviceUrl.replace(regExp, action.replacement);
-								mw.config.set(key, serviceUrl);
+                                serviceUrl = serviceUrl.replace(regExp, action.replacement);
+                                mw.config.set(key, serviceUrl);
+                                // Pass the override URLs configurations to the parent mw object so that it's client
+                                // URLs would be updated too.
+                                if(mw.config.get( 'EmbedPlayer.IsFriendlyIframe') ){
```

Comment:
we must have some property to enable or disable this, because not all KES on customer side have latest code . (default is disable)